src/localplayer.cpp

```diff
@@ -663,7 +663,8 @@ void LocalPlayer::applyControl(float dtime, Environment *env)
 	// Slip on slippery nodes
 	const INodeDefManager *nodemgr = env->getGameDef()->ndef();
 	Map *map = &env->getMap();
-	const ContentFeatures &f = nodemgr->get(map->getNodeNoEx(getStandingNodePos()));
+	const ContentFeatures &f = nodemgr->get(
+			map->getNodeNoEx(floatToInt(getPosition() - v3f(0, 0.05f * BS, 0), BS)));
```

Comment:
Would it help to use `0.0f` instead of `0`? The `0` might be seen as a double not a float. I would prefer `0.0f`.